tests/drivers/w5500/Makefile.in

```diff
@@ -0,0 +1,14 @@
+BOARD_INSUFFICIENT_MEMORY := \
```

Comment:
This file should be removed

Reply:
Ok, I have modified the Makefile.ci and removed the Makefile.in... If I had used my brain I could have come up with the idea that the Makefile.ci is the relevant file :-)